src/Place/GeoCoordinatesCommandHandler.php

```diff
@@ -46,11 +54,19 @@ public function __construct(
     public function handleUpdateGeoCoordinatesFromAddress(UpdateGeoCoordinatesFromAddress $updateGeoCoordinates)
     {
         $coordinates = $this->geocodingService->getCoordinates(
-            $this->addressFormatter->format(
+            $this->defaultAddressFormatter->format(
                 $updateGeoCoordinates->getAddress()
             )
         );
 
+        if ($coordinates === null) {
```

Comment:
Did you consider looping through the available address formatters until one results in valid coordinates / until there are no formatters left?

Reply:
I did base did approach on the strategy pattern. The returned result does indicate which formatter to use, although there is some duplication now.